app/assets/stylesheets/users.scss

```diff
@@ -75,15 +87,85 @@
   width: 100%;
 }
 
-.users-social-links {
-  height: 50px;
-  margin-left: 5px;
-  margin-right: 5px;
-  padding-top: 10px;
-  width: 40px;
+
+.auth-btns-container {
```

Comment:
Add center aligning to this outer div. Make sure button moves to next line for mobile displays and still remains in center. (Not sure if this can be pulled with display:flex)

Reply:
@aayush-05  **align-items: center** will do the job.

Reply:
Was just going by the screenshots; they are left aligned there. Haven't pulled it on local yet to check. Please ensure the css is not getting overridden